gdx/src/com/badlogic/gdx/utils/Queue.java

```diff
@@ -142,8 +141,7 @@ public T removeFirst () {
 
 		final T result = values[head];
 		values[head] = null;
-		head++;
```

Comment:
I have assumed so. My argument is that:
```
this.head++;
if(this.head == values.length) ...
```
requires two reads from the object, while:
```
if(++this.head == values.length) ...
```
has to read just once, which could be slightly faster, in some circumstances.

But it is not a big deal either way. I'll wait for core contributors to decide.